tools/bumpVersion.sh

```diff
@@ -7,38 +7,30 @@ readonly TOP
 
 readonly PACKAGE_JSON="$TOP/package.json"
 
-# Read the version from package.json as the "current version"
-CURRENT_VERSION="$(grep -E '"version": "([0-9\.]+)"' "$PACKAGE_JSON" | awk -F\" '{print $4}')"
+# Save the current package.json version to a variable
+CURRENT_VERSION="$(jq -r '.version' < "$PACKAGE_JSON")"
 readonly CURRENT_VERSION
 
 # Parse the version into major/minor/patch
-readonly MAJOR="${CURRENT_VERSION%%.*}"
+readonly _MAJOR="${CURRENT_VERSION%%.*}"
 readonly REMAINING="${CURRENT_VERSION#*.}"
 readonly MINOR="${REMAINING%%.*}"
 readonly PATCH="${CURRENT_VERSION##*.}"
 
-# Increment the patch version
-NEW_MAJOR="$MAJOR"
-NEW_MINOR="$MINOR"
-NEW_PATCH="$((PATCH + 1))"
-
-# If patch is > 99, bump minor and reset patch to 0
-if [[ "$NEW_PATCH" -gt 99 ]] ; then
-    NEW_PATCH=0
-    NEW_MINOR="$((NEW_MINOR + 1))"
-fi
-
-# If minor is > 99, bump major and reset minor to 0
-if [[ "$NEW_MINOR" -gt 99 ]] ; then
-    NEW_MINOR=0
-    NEW_MAJOR="$((NEW_MAJOR + 1))"
+if [[ "$PATCH" != "99" ]]; then
+    npm --no-git-tag-version version patch
+elif [[ "$MINOR" != "99" ]]; then
+    npm --no-git-tag-version version minor
+else
+    npm --no-git-tag-version version major
 fi
```

Comment:
NAB I find this easier to reason about, and I think it will produce the same result:

```bash
if [[ "$MINOR" -eq 99 ]] ; then
    npm --no-git-tag-version version major
elif [[ "$PATCH" -eq 99 ]] ; then
    npm --no-git-tag-version version minor
else
    npm --no-git-tag-version version patch
fi
```